src/index.rs

```diff
@@ -1133,18 +1133,19 @@ mod tests {
         assert!(new2 > new1);
     }
 
+    const CREATE: i32 = 1;
+    const UPDATE: i32 = 2;
+    const DELETE: i32 = 3;
+    const MOVE: i32 = 5;
```

Comment:
Why `MOVE` is `5`?

Actually, it's better to use `enum` to avoid mistakes with numbers.